src/app/grama-niladhari/components/dashboard/appoinments/appoinments.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+/* tslint:disable:no-unused-variable */
```

Comment:
don't disable warnings

Reply:
It was automatically commented. so what should I do in my side?

Reply:
Huh okay ignore it then leave it commented